component/publisher-client/src/test/java/org/wso2/am/analytics/publisher/DefaultResponseMetricBuilderTestCase.java

```diff
@@ -89,6 +89,7 @@ public void testMetricBuilder() throws MetricCreationException, MetricReportingE
                 .addAttribute(Constants.CORRELATION_ID, "1234-4567")
                 .addAttribute(Constants.KEY_TYPE, "prod")
                 .addAttribute(Constants.API_ID, "9876-54f1")
+                .addAttribute(Constants.API_TYPE, "HTTP")
```

Comment:
API_TYPE => REST. Otherwise, it might give a false impression for users reading the test cases.